test/module/irohad/ametsuchi/ametsuchi_mocks.hpp

```diff
@@ -167,7 +170,7 @@ namespace iroha {
     expected::Result<std::unique_ptr<MutableStorage>, std::string>
     createMockMutableStorage() {
       return expected::makeValue<std::unique_ptr<MutableStorage>>(
-          std::move(std::make_unique<MockMutableStorage>()));
+          std::make_unique<MockMutableStorage>());
```

Comment:
Why `std::move` here is removed?

Reply:
Because the argument is already a rvalue.